setupTests.js

```diff
@@ -0,0 +1,7 @@
+import Enzyme from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16';
+import chai from 'chai'
+import chaiEnzyme from 'chai-enzyme'
+
+chai.use(chaiEnzyme())
+Enzyme.configure({ adapter: new Adapter() })
```

Comment:
I'm not sure this is entirely necessary. Can we do something a bit simpler, closer to [this example](https://github.com/facebook/jest/tree/master/examples/react) from Facebook?

Reply:
I included chai-enzyme  for the style assertions "to.have.style" as it seemed easiest.

I started with that approach and then saw that react-addons-test-utils was deprecated and when I went to https://reactjs.org/docs/test-utils.html to see what was needed to migrate they recommended Enzyme. I also needed to use mount from Enzyme for Full Dom Rendering and it seemed cleaner than TestUtils.renderIntoDocument(). 

I am open to suggestions but it I managed to get this working and it took longer than I expected already.

Reply:
Okay, no problem. I'll merge this now and publish it on npm. Sometime in the next few weeks I'll refactor and publish again. Cheers for the work and happy holidays!